### PR TITLE
Introduce the namespace for the library

### LINKED ISSUE #68: Introduce the namespace for the library
Move the whole library implementation under the namespace not to interfere with global namespace

### PR DESCRIPTION
**Pull request by Semyon1104**
close #68

### CODE REVIEW

**Comment by Semyon1104** [comment]
This branch is branched off from a branch that hasn't been merged yet - Move alexnet_src.cpp and reader_img.cpp to app directory

**Comment by aobolensk** [comment]
> This branch is branched off from a branch that hasn't been merged yet - Move alexnet_src.cpp and reader_img.cpp to app directory

You may update your branch now. https://github.com/embedded-dev-research/itlab_2023/pull/111 has been merged

File: include/layers/ConvLayer.hpp [hunk 1]
```diff
@@ -102,4 +104,5 @@ class ConvImpl : public LayerImpl<ValueType> {
     }
     return outputvec;
   }
-};
\ No newline at end of file
+};
+}  // namespace itlab_2023
```
Comment (aobolensk) [inline include/layers/ConvLayer.hpp:108]:
```suggestion
}  // namespace itlab_2023

```

File: include/layers/Layer.hpp [hunk 2]
```diff
@@ -34,3 +36,4 @@ class LayerImpl {
   Shape inputShape_;
   Shape outputShape_;
 };
+}  // namespace itlab_2023
```
Comment (aobolensk) [inline include/layers/Layer.hpp:39]:
```suggestion
}  // namespace itlab_2023

```

File: src/layers/ConvLayer.cpp [hunk 3]
```diff
@@ -92,4 +94,5 @@ void ConvolutionalLayer::run(const Tensor& input, Tensor& output,
       throw std::runtime_error("No such type");
     }
   }
-}
\ No newline at end of file
+}
+}  // namespace itlab_2023
```
Comment (aobolensk) [inline src/layers/ConvLayer.cpp:98]:
```suggestion
}  // namespace itlab_2023

```

**Review by aobolensk — CHANGES_REQUESTED** [review]

**Review by aobolensk — APPROVED** [review]

**Pull request merged**